log/log.go

```diff
@@ -115,9 +120,9 @@ func Error(ctx context.Context, fields map[string]interface{}, format string, ar
 
 		if ctx != nil {
 			entry = entry.WithField("req_id", extractRequestID(ctx))
-			identity_id, err := extractIdentityID(ctx)
+			identityID, err := extractIdentityID(ctx)
```

Comment:
Finally somebody reads the go analysis outputs ;) But really, those fixes should be in their own PR.